datafusion/core/src/datasource/avro_to_arrow/schema.rs

```diff
@@ -158,6 +159,8 @@ fn schema_to_field_with_props(
         AvroSchema::TimeMicros => DataType::Time64(TimeUnit::Microsecond),
         AvroSchema::TimestampMillis => DataType::Timestamp(TimeUnit::Millisecond, None),
         AvroSchema::TimestampMicros => DataType::Timestamp(TimeUnit::Microsecond, None),
+        AvroSchema::LocalTimestampMillis => todo!(),
```

Comment:
Type conversion from `LocalTimestamp` is not trivial. So leave them as `todo!()` now.